internal/admin/dashboard/static/css/dashboard.css

```
@@ -2173,3 +2362,469 @@ body.conversation-drawer-open {
+    justify-content: center;
+    width: 28px;
+    height: 28px;
+    border-radius: 8px;
+    background: var(--bg);
+    color: var(--text-muted);
+}
+
+.ep-node-icon svg {
+    width: 15px;
+    height: 15px;
+    stroke: currentColor;
```

Stylelint (17.5.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)